spec/spec_helper.rb

```diff
@@ -16,4 +15,4 @@
 
 RSpec.configure do |config|
```

Comment:
This empty RSpec.configure is necessary?

Reply:
It exists only to indicates where really are the options...